Jenkinsfile

```diff
@@ -16,102 +16,10 @@
-                    }
-                    stages {
-                        stage('Docker Build') {
-                            steps {
-                                edgeXDockerLogin(settingsFile: env.MVN_SETTINGS)
-
-                                script {
-                                    image_amd64 = docker.build('docker-device-bacnet-c', '-f scripts/Dockerfile.alpine-3.9 .')
-                                }
-                            }
-                        }
-
-                        stage('Docker Push') {
-                            when { expression { edgex.isReleaseStream() } }
-                            steps {
-                                script {
-                                    docker.withRegistry("https://${env.DOCKER_REGISTRY}:10004") {
-                                        image_amd64.push(env.GIT_COMMIT)
-                                        image_amd64.push('1.1.0')
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-
-                stage('arm64') {
-                    agent {
-                        label 'ubuntu18.04-docker-arm64-4c-2g'
-                    }
-                    stages {
-                        stage('Docker Build') {
-                            steps {
-                                edgeXDockerLogin(settingsFile: env.MVN_SETTINGS)
-
-                                script {
-                                    image_arm64 = docker.build('docker-device-bacnet-c-arm64', '-f scripts/Dockerfile.alpine-3.9 .')
-                                }
-                            }
-                        }
-
-                        stage('Docker Push') {
-                            when { expression { edgex.isReleaseStream() } }
-                            steps {
-                                script {
-                                    docker.withRegistry("https://${env.DOCKER_REGISTRY}:10004") {
-                                        image_arm64.push(env.GIT_COMMIT)
-                                        image_arm64.push('1.1.0')
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-            }
-        }
-
-        stage('Clair Image Scan') {
-            when { expression { edgex.isReleaseStream() } }
-            steps {
-                edgeXClair("${env.DOCKER_REGISTRY}:10004/docker-device-bacnet-c:1.1.0")
-                edgeXClair("${env.DOCKER_REGISTRY}:10004/docker-device-bacnet-c-arm64:1.1.0")
-            }
-        }
-    }
-
-    post {
-        always {
-            edgeXInfraPublish()
-        }
-    }
-}
+edgeXBuildDocker (
+    project: 'device-bacnet-c',
+    dockerFilePath: './scripts/Dockerfile.alpine-3.9'
+)
 
 def loadGlobalLibrary(branch = '*/master') {
```

Comment:
let's work on removing this code from the Jenkinsfile. I am working with the LF on that right now.